src/js/apps/globals/nav_app.js

```diff
@@ -170,4 +175,40 @@ export default App.extend({
 
     return navView;
   },
+  onSearch(navView) {
+    const resultsCollection = Radio.request('entities', 'searchPatients:collection');
+
+    const patientSearchModal = new PatientSearchModal({
+      resultsCollection,
+    });
+
+    this.listenTo(patientSearchModal, {
+      'update:query': state => {
+        const query = state.get('query');
+
+        if (query.length < 2) {
+          if (query.length === 0) resultsCollection.reset();
+          return;
+        }
+
+        Radio.request('entities', 'fetch:searchPatients', query).then(results => {
+          resultsCollection.reset(results.models);
+        });
+      },
+      'item:select': ({ model }) => {
+        Radio.trigger('event-router', 'patient:dashboard', model.get('_patient'));
+        patientSearchModal.destroy();
+      },
+      'close': () => {
```

Comment:
Isn't `destroy` always called?  Is `close` really needed?  If so that seems like a bug?